jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildImageSteps.java

```diff
@@ -229,7 +229,11 @@ public void run()
                   buildImageStep);
 
           timer2.lap("Running push new image");
-          pushImageStep.getFuture().get();
+          try {
+            pushImageStep.getFuture().get();
+          } catch (ExecutionException ex) {
+            ex.printStackTrace();
```

Comment:
is this intended? Just print the stacktrace?

Reply:
Oh right forgot to take this out.